201-logic-app-xslt-with-params/azuredeploy.json

```diff
@@ -0,0 +1,188 @@
+      }
+    },
+    "transformFunctionName": {
+      "type": "string",
+      "defaultValue": "transformfunction",
+      "metadata": {
+        "description": "The name of the function."
+      }
+    },
+    "integrationAccountName": {
+      "type": "string",
+      "defaultValue": "IntegrationAccount",
+      "minLength": 1,
+      "maxLength": 80,
+      "metadata": {
+        "description": "Name of the Integration Account."
+      }
+    },
+    "logicAppName": {
+      "type": "string",
+      "defaultValue": "XSLTWithParams",
+      "minLength": 1,
+      "maxLength": 80,
+      "metadata": {
+        "description": "Name of the Logic App."
+      }
+    },
+    "logicAppLocation": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "allowedValues": [
+        "[resourceGroup().location]",
+        "australiaeast",
+        "australiasoutheast",
+        "brazilsouth",
+        "centralus",
+        "eastasia",
+        "eastus",
+        "eastus2",
+        "japaneast",
+        "japanwest",
+        "northcentralus",
+        "northeurope",
+        "southcentralus",
+        "southeastasia",
+        "westeurope",
+        "westus"
+      ],
+      "metadata": {
+        "description": "Location of the Logic App."
+      }
+    }
+  },
+  "variables": { },
+  "resources": [
+    {
+      "properties": { },
+      "sku": {
+        "name": "Free"
+      },
+      "name": "[parameters('integrationAccountName')]",
+      "type": "Microsoft.Logic/integrationAccounts",
+      "location": "[parameters('logicAppLocation')]",
+      "apiVersion": "2015-08-01-preview"
+    },
+    {
+      "properties": {
+        "mapType": "xslt",
+        "content": "<?xml version=\"1.0\" encoding=\"utf-16\"?><xsl:stylesheet xmlns:xsl=\"http://www.w3.org/1999/XSL/Transform\" xmlns:input=\"urn:my-input-variables\" xmlns:msxsl=\"urn:schemas-microsoft-com:xslt\" xmlns:var=\"http://schemas.microsoft.com/BizTalk/2003/var\" exclude-result-prefixes=\"msxsl var s0 userCSharp\" version=\"1.0\" xmlns:ns0=\"http://Integration.Destination\" xmlns:s0=\"http://Integration.Source\" xmlns:userCSharp=\"http://schemas.microsoft.com/BizTalk/2003/userCSharp\"><xsl:import href=\"https://az818438.vo.msecnd.net/functoids/functoidsscript.xslt\" /><xsl:output omit-xml-declaration=\"yes\" method=\"xml\" version=\"1.0\" />  <xsl:param name=\"input:X\"></xsl:param><xsl:param name=\"input:Y\"></xsl:param><xsl:template match=\"/\"><xsl:apply-templates select=\"/s0:Root\" /></xsl:template><xsl:template match=\"/s0:Root\"><ns0:Root><Sum><xsl:value-of select=\"userCSharp:MathAdd($input:X, $input:Y)\" /></Sum><Diff><xsl:value-of select=\"userCSharp:MathSubtract($input:X, $input:Y)\" /></Diff><Product><xsl:value-of select=\"userCSharp:MathMultiply($input:X, $input:Y)\" /></Product><Division><xsl:value-of select=\"userCSharp:MathDivide($input:X, $input:Y)\" /></Division></ns0:Root></xsl:template></xsl:stylesheet>",
```

Comment:
should this be hardcoded?  https://az818438.vo.msecnd.net/functoids/functoidsscript.xslt